bedrock/mozorg/templates/mozorg/home/includes/fundraiser-eoy2018.html

```diff
@@ -0,0 +1,69 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+<aside class="c-fundraiser" id="fundraiser">
+  <button type="button" class="c-fundraiser-close" id="fundraiser-close"><span>{{ _('Close') }}</span></button>
+  <div class="content mzp-l-content">
+    <h1 class="c-fundraiser-title">
+      {{ _('We all love the web. Join Mozilla in defending it.') }}
+    </h1>
+
+    <div class="c-fundraiser-content">
+      <p>
+      {% if LANG == 'en-US' %}
```

Comment:
Could we show this string for all English locales?
```suggestion
      {% if LANG.startswith('en-') %}
```